dragons-nft-scripts/src/index.js

```diff
@@ -0,0 +1,293 @@
+const { CLValueBuilder, Keys, CLPublicKey, CLPublicKeyType } = require("casper-js-sdk");
+const { CEP47Client, utils, constants } = require("casper-cep47-js-client");
+
+// const NODE_ADDRESS = 'http://3.143.158.19:7777/rpc';
+const NODE_ADDRESS = 'http://localhost:40101/rpc';
+// const EVENT_STREAM_ADDRESS = 'http://3.143.158.19:9999/events';
+const EVENT_STREAM_ADDRESS = 'http://localhost:60101/events';
+const INSTALL_PAYMENT_AMOUNT = '200000000000';
+const MINT_ONE_PAYMENT_AMOUNT = '2000000000';
+const MINT_COPIES_PAYMENT_AMOUNT = '100000000000';
+const BURN_ONE_PAYMENT_AMOUNT = '12000000000';
+// const CHAIN_NAME = 'integration-test';
+const CHAIN_NAME = 'casper-net-1';
+const WASM_PATH = "/home/ziel/workspace/casperlabs/ipwe-nft-contract/target/wasm32-unknown-unknown/release/ipwe-nft-contract.wasm";
```

Comment:
I don't have this file path. Some general what this `index.js` is and how to operate it would be appreciated.

Reply:
I'm moving `dragons-nft-scripts/` to https://github.com/casper-ecosystem/casper-cep47-js-client/blob/feature/1.3.0/test/basic.ts, so It can be removed from here soon :)

Reply:
Thank you!